src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerEnclaveProvider.java

```diff
@@ -57,19 +78,152 @@
 
     /**
      * Returns the enclave session
-     * @return
-     *         the enclave session
+     * 
+     * @return the enclave session
      */
     EnclaveSession getEnclaveSession();
 }
 
 
 abstract class BaseAttestationRequest {
     protected PrivateKey privateKey;
+    // Static byte[] for VSM ECDH
+    protected static byte ECDH_MAGIC[] = {0x45, 0x43, 0x4b, 0x33, 0x30, 0x00, 0x00, 0x00};

```

Comment:
variable name seems to suggest this line / ENCLAVE_LENGTH were meant to be final variables?

Reply:
also move position of []